src/components/RightSidebar/Participants/ParticipantsSearchResults/ParticipantsSearchResults.vue

```diff
@@ -46,6 +46,24 @@
 				@click="handleClickParticipant" />
 		</template>
 
+		<!-- integrations -->
+		<template v-if="integrations.length !== 0">
+			<AppNavigationCaption :title="t('spreed', 'Integrations')" />
+			<ul>
+				<Button v-for="(integration, index) in integrations"
+					:key="'integration' + index"
+					type="tertiary-no-background"
```

Comment:
I would have prefered the avatar style with the circle permanently around it.
But good enough for now